core/tests/protractor.conf.js

```diff
@@ -315,7 +315,7 @@ exports.config = {
 
   // Selector for the element housing the angular app - this defaults to
   // body, but is necessary if ng-app is on a descendant of <body>.
-  rootElement: 'body',
+  // rootElement: 'body',
```

Comment:
Srijan wanted to remove this one, as this is redundant. Default value is also `body`

Reply:
sg, let's remove this!

Reply:
done